Makefile

```diff
@@ -5,13 +5,13 @@ cs_dry_run: ## Display CS violations without fixing it
 	./vendor/bin/php-cs-fixer fix --verbose --dry-run
 
 spec_diff: ## detect diff between spec and SDK
-	CI_TEST=1 vendor/bin/jane-openapi generate --config-file=.jane-openapi.php && diff generated_ci generated
+	CI_TEST=1 vendor/bin/jane-openapi generate --config-file=.jane-openapi.php && vendor/bin/php-cs-fixer fix && diff generated_ci generated
 
 test: ## Run test suite
 	./vendor/bin/simple-phpunit
 
 jane: ## Generate the SDK
-	rm -rf generated/* && vendor/bin/jane-openapi generate --config-file=.jane-openapi.php
+	rm -rf generated/* && vendor/bin/jane-openapi generate --config-file=.jane-openapi.php && vendor/bin/php-cs-fixer fix
```

Comment:
The deletion of source file is already done by `clean-generated` command option (which is `true` by default)